tensorflow/contrib/rnn/python/kernel_tests/rnn_cell_test.py

```diff
@@ -1671,5 +1671,64 @@ def cell():
     self.assertAllClose(expected_c, actual_c, 1e-5)
     self.assertAllClose(expected_h, actual_h, 1e-5)
 
+class MLSTMCellTest(test.TestCase):
+  """Test the MLSTM cell."""
+
+  def _cell_output(self, cell):
+    """Calculates cell output."""
+
+    with self.test_session() as sess:
+      init = init_ops.constant_initializer(0.5)
+      with variable_scope.variable_scope("root",
+                                         initializer=init):
+        x = array_ops.zeros([1, 2])
+        c0 = array_ops.zeros([1, 2])
+        h0 = array_ops.zeros([1, 2])
+
+        state0 = rnn_cell.LSTMStateTuple(c0, h0)
+
+        xout, sout = cell()(x, state0)
```

Comment:
This kind of confusing, how about rename this function to create_cell, since it is not a cell instance but a function to create cell.

Reply:
Hello, thanks for reviewing my pull request.

Just to clarify, I should change the _cell_output() to create_cell()?